AISKU/package.json

```diff
@@ -29,23 +29,23 @@
         "nightwatch:done": "curl http://localhost:8000/_done"
     },
     "devDependencies": {
+        "@microsoft/applicationinsights-rollup-plugin-uglify3-js": "1.0.0",
         "@microsoft/applicationinsights-rollup-es3" : "1.1.3",
         "@types/qunit": "^2.5.3",
         "@types/sinon": "4.3.3",
         "qunit": "^2.9.1",
         "sinon": "^7.3.1",
-            "chromedriver": "^2.45.0",
+        "chromedriver": "^2.45.0",
         "finalhandler": "^1.1.1",
-        "grunt": "1.0.1",
+        "grunt": "^1.3.0",
         "grunt-contrib-qunit": "^3.1.0",
-        "grunt-contrib-uglify": "3.1.0",
-        "grunt-ts": "^6.0.0-beta.15",
+        "grunt-ts": "^6.0.0-beta.22",
         "grunt-tslint": "^5.0.2",
         "globby": "^11.0.0",
-        "rollup": "^0.66.0",
-        "rollup-plugin-node-resolve": "^3.4.0",
-        "rollup-plugin-replace": "^2.1.0",
-        "rollup-plugin-uglify": "^6.0.0",
+        "@rollup/plugin-commonjs": "^15.1.0",
```

Comment:
Are these 3 packages different? eg. `rollup-plugin-node-resolve` vs `@rollup/plugin-node-resolve`, is the package name changed?

Reply:
This is part of rollup moving their packages, so the latest version need to use the new package location